Automattic/wp-calypso
# A4A: add woo product-filters data

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/882

## Proposed Changes

As a part of adding new product we need to make sure we have proper product name and description within A4A. This PR adds description to the A4A marketplace as well as Product lighbox:

**Marketplace**
<img width="358" alt="Screenshot 2024-10-29 at 6 19 39 PM" src="https://github.com/user-attachments/assets/70f4162b-1ab3-484c-84a4-349c72ccc123">

**Lighbox**
<img width="941" alt="Screenshot 2024-10-29 at 6 19 31 PM" src="https://github.com/user-attachments/assets/713ccafb-d892-403c-bd1d-35b0df0f8ca9">

**Checkout**
<img width="749" alt="Screenshot 2024-10-29 at 6 19 52 PM" src="https://github.com/user-attachments/assets/90157fc3-31ea-4964-b665-631f6af49d06">

## Why are these changes being made?



## Testing Instructions



- Change billing scheme. `wp jetpack-start partner-key-billing-scheme-override add [Partner Key ID] 877` 
- Navigate to `agencies.automattic.com/marketplace/products` and check that you see the Woo Product Filters product. Check the descriptions and purchase flow.

## Pre-merge Checklist
